src/panels/config/ha-panel-config.ts

```diff
@@ -217,6 +225,13 @@ class HaPanelConfig extends HassRouterPage {
             /* webpackChunkName: "panel-config-integrations" */ "./integrations/ha-config-integrations"
           ),
       },
+      lovelace: {
```

Comment:
I think that we used to always load it, that speeds up our load times significantly. Why not do that ?

Reply:
I don't get it? This the config page, we don't want to load that always right?